jenkins-scripts/dsl/subt.dsl

```diff
@@ -166,9 +166,17 @@ all_supported_distros.each { distro ->
     long_run_job.with
     {
       scm {
-        hg('https://bitbucket.org/osrf/subt') {
-          branch('default')
-          subdirectory('subt')
+        git {
+          remote {
+            github("osrf/subt", 'https')
+            branch('${BRANCH}')
```

Comment:
the old one is always running on `default`; should this be `master` instead of `${BRANCH}`?

Reply:
good catch! fixed in ce991a6